gulp/js/router.js

```diff
@@ -3,8 +3,8 @@ app.routers.DefaultRouter = Backbone.Router.extend({
   routes: {
     "":                             			"index",
     "?*queryString":                			"index",
-    "transcripts/:instituion/:id":  			"transcriptEdit",
-    "transcripts/:instituion/:id?*queryString": "transcriptEdit",
+    "transcripts/:institution/:collection/:id":  			"transcriptEdit",
+    "transcripts/:institution/:collection/:id?*queryString": "transcriptEdit",
```

Comment:
So you're already applying the changes here, yeah?

Reply:
Yes, I copied the changes from your fork. Putting them in gulp/js/router.js will ensure that they're kept whenever the file is rebuilt.